test/Protocol.hs

```diff
@@ -19,6 +20,14 @@ import Network.Pusher.Protocol
   , Users(..)
   )
 
+newtype TestChannel = TestChannel Channel deriving (Show)
+
+instance Arbitrary TestChannel where
+  arbitrary = do
+    let testChannelType = elements [Public, Private, Presence]
+        channel = Channel <$> testChannelType <*> (T.pack <$> arbitrary)
```

Comment:
Minor stylistic thing: I've followed a style of no column alignment and 2 character indents, and I'd like to keep that consistent. Would you mind changing this to:

```
let
  testChannelType = elements [Public, Private, Presence]
  channel = Channel <$> testChannelType <*> (T.pack <$> arbitrary)
```